### ISSUE TITLE
assembling for ti994a

### ISSUE THREAD

**Issue by J03M4N** [nanochess/CVBasic#16]
trying to assemble for ti994a using xdt99 and keep getting a bank size error:
    raise AsmError(f'Bank number too large, must be less than { Symbols.BANK_ALL
 }')
Not sure if I am doing something wrong, I read the xdt99 docs. tried with space_attack.bas and gorilla.bas

**Comment by nanochess** [comment]
Hi.

The latest CVBasic version requires new compilation instructions:

cvbasic --ti994a space_attack.bas space_attack.a99
xas99.py -b -R space_attack.a99
linkticart.py space_attack.bin space_8.bin "SPACE"

I think these have been updated, but maybe you have the older README.

I just could compile without any problem.

**Comment by J03M4N** [comment]
Cool! Thank you so much, it worked. Turns out that I had an older version of python. Sorry for my stupidity, but I still needed help because I have cvbasic 0.7.1, but the readme doesn't include the new assembly instructions, it just says to use the -R option, so thanks for your assistance, you're very kind.
I created an RPK and loaded it up in an older version of MAME (I think newer versions don't require an RPK, not sure though) , and it works!
